app.yaml

```diff
@@ -49,6 +49,11 @@ handlers:
   # NB: not minified. TODO(sll): fix.
   static_dir: core/templates/dev/head/css
   secure: always
+  # Serve only the js files in this directory as static files.
```

Comment:
Ditto here compared to my comment in #2356. We should be clear in the comments about what these patterns are doing.